setup.py

```diff
@@ -18,7 +18,7 @@
 
 
 requirements = [
-    'graphene-sqlalchemy>=2.1.0,<3',
+    'graphene-sqlalchemy>=2.1.0',
```

Comment:
This should be `graphene-sqlalchemy>=3` because the `GraphQLResolveInfo` was introduced in v3 and is not in 2.x